DependencyInjection/HttplugExtension.php

```diff
@@ -205,4 +219,25 @@ private function configureAuthentication(ContainerBuilder $container, array $con
                 ->addArgument(new Reference($authServiceKey));
         }
     }
+
+    /**
+     * Resolve the plugin enabled status if it is 'auto'.
+     *
+     * Returns false if plugin has no auto status allowed.
+     *
+     * @param string $name
+     * @param array  $pluginConfig
+     *
+     * @return bool
+     */
+    private function resolvePluginStatus($name, array $pluginConfig)
```

Comment:
This method is not necessary ATM. I started to add this auto, true, false enabled config, but I realized that even in auto mode as default, it would be a BC break for already existing configuration. That's why I added back the plugins dependency. When we release 2.0, we can remove the plugins dep, add the proposed auto default value. Until then, this method is not necessary.